libbeat/processors/add_docker_metadata/add_docker_metadata.go

```diff
@@ -22,13 +22,13 @@ package add_docker_metadata
 import (
 	"fmt"
 	"os"
-	"path/filepath"
+	"regexp"
 	"strings"
 	"time"
 
 	"github.com/pkg/errors"
 
-	"github.com/elastic/gosigar/cgroup"
+	"github.com/elastic/beats/v7/libbeat/metric/system/cgroup"
 
```

Comment:
Nit. Move this import to the group of imports below.
```suggestion
	"github.com/elastic/beats/v7/libbeat/metric/system/cgroup"
```